api/src/pcapi/connectors/big_query/queries/offerer_stats.py

```diff
@@ -29,12 +29,13 @@ def raw_query(self) -> str:
             `{settings.BIG_QUERY_NOTIFICATIONS_TABLE_BASENAME}.{DAILY_CONSULT_PER_OFFERER_LAST_180_DAYS_TABLE}`
         WHERE
             offerer_id = @offerer_id
-            """
+        limit 180
```

Comment:
[détail] Je n'ai pas de préférence pour les majuscules ou les minuscules. Mais j'ai une (nette) préférence pour l'homogénéité. ;)

```suggestion
        LIMIT 180
```

Idem pour l'autre requête plus bas.

Reply:
Bien vu, done